docs/pages/enroll-resources/kubernetes-access/rancher-integration.mdx

```diff
@@ -0,0 +1,269 @@
+the general approach remains the same. Rancher will return users from the configured external source during role binding and access configuration.
+
+## Step 2/4. Enroll clusters to Teleport
+
+This step connects your Rancher-managed clusters to Teleport for centralized access control:
+
+To enroll your cluster(s), see the official [Teleport Kubernetes Access documentation](/docs/kubernetes-access/getting-started/)
+for detailed enrollment instructions.
+
+## Step 3/4. Configure cluster access in Teleport
+
+Now we'll define Teleport roles that map to Rancher's RBAC permissions:
+
+We'll create a Teleport role that authenticates users to the cluster
+as a member of the `github_team://7415717` group subject assigned the Rancher `cluster-member` Role.
+
+In the previous section, we discovered the built-in Cluster Member role authorizing members
+of this group to perform read-only operations at the cluster level. This specific role:
+- Enables the user to access cluster nodes
+- Denies the user permissions to create, modify or delete resources (e.g. can't deploy workloads)
+
+For more information on permissions granted by the Cluster Member role, see the following guide on
+[Cluster and Project Roles](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/cluster-and-project-roles)
+
+### Define a Teleport Role
+
+Create a file called `kube-access.yaml` with the following content:
+
+```yaml
+kind: role
+version: v7
+metadata:
+  name: kube-access
+spec:
+  allow:
+    kubernetes_groups: ["github_team://7415717"]
+    kubernetes_labels:
+      "*": "*"
+    kubernetes_resources:
+      - kind: "*"
+        namespace: "*"
+        name: "*"
+```
+
+In this role, we have defined the following allow rules:
+- `kubernetes_groups`: Authenticates the user to your Kubernetes cluster as a member of the Kubernetes group subject 
+`github_team://7415717`, authorizing members of this group with the Cluster Member Role permissions
+
+### Create the Role
+
+Once you have finished configuring the kube-access role, create it using the following command:
+
+```code
+$ tctl create -f kube-access.yaml
+```
+
+### Assign the Role
+
+Assign the kube-access role to your Teleport user by running the appropriate commands for your authentication provider:
+
+```code
+$ tctl users update <username> --set-roles=kube-access
+```
+
+## Step 4/4. Access and verify resources
+
+Finally, we'll test the integration by accessing resources through Teleport:
+
+You've now configured the Teleport Kubernetes Service to give your Teleport user
+access to the Rancher cluster as a user with the built-in Rancher Cluster Member Role. In this step,
+we will authenticate to your Rancher cluster via Teleport and test our new access controls.
+
+### Log in to Teleport
```

Comment:
```suggestion
Log in to Teleport:
```
Since these H3-level sections are very short, I think we can turn the headings into paragraphs